src/components/navigation-header/components/home-page-content/index.tsx

```diff
@@ -54,7 +54,7 @@ const HomePageHeaderContent = () => {
 					src={require('../../img/logo-white.svg?include')}
 				/>
 			</div>
-			<div className="g-hide-on-mobile g-hide-on-tablet">
+			<div className={sharedNavStyles.leftSideDesktopOnlyContent}>
```

Comment:
In order to fit the Registry link, I've done a very light refactor allow us to use a more flexible breakpoint for switching between the "mobile" and "desktop" modes of `navigation-header`.

This className change is one part of that. In general, where necessary to achieve this type of flexibility, I've replaced uses of `g-hide-on-*` as those use fixed breakpoints.

---

> **Note**: 💭 This might not feel ideal. I think a key note here is that I think, long term, this type of flexibility will be essential if we want the site to genuinely look and function well at any viewport size (rather than have it be optimized only at specific, somewhat arbitrary "mobile", "tablet", and "desktop" breakpoints).

> 📌 Future thought: Even better, from my perspective, would be to find a way to have "dynamic breakpoints" - this way, menus would collapse when necessary to prevent overflow rather than at specific pixel width breakpoints. However, that type of enhancement feels well outside the scope of this PR.